sdcm/cluster.py

```diff
@@ -64,7 +64,7 @@
 TEST_DURATION = 60
 # max limit of coredump file can be uploaded(5 GB)
 COREDUMP_MAX_SIZE = 1024 * 1024 * 1024 * 5
-IP_SSH_CONNECTIONS = 'public'
+IP_SSH_CONNECTIONS = 'private'
```

Comment:
dangerous :)

Reply:
I saw it too, but I think it's better like this.